### PR TITLE
Skipping local webservice test from build agents

### PR DESCRIPTION
**Pull request by heemanshusuri**

### CODE REVIEW

File: .azure-pipelines/scripts/validate_samples.R [hunk 1]
```diff
@@ -24,11 +24,21 @@ getPathLeaves <- function(path){
...
-validate_samples <- function(directory) {
-
+validate_samples <- function(args) {
+  directory = args[1]
   sample_dirs = getPathLeaves(directory)
 
+  skip_tests = c()
+
+  if (length(args) > 1) {
+    skip_tests <- args[2:length(args)]
```
Comment (ninghu) [inline .azure-pipelines/scripts/validate_samples.R]: in case of multiple tests, i'd separate by semicolon.

Comment (heemanshusuri) [reply .azure-pipelines/scripts/validate_samples.R]: you can pass them as separate argument. it looks cleaner

Comment (ninghu) [reply .azure-pipelines/scripts/validate_samples.R]: what if we need to introduce another argument for other purpose in the future? that very likely will be the case when add validation for vignettes (i.e. add another exclude list for vignettes).  Combing them into a single argument can at least make sure the existing logic and usage won't be changed when adding a new argument. Just my 2 cents.

Comment (heemanshusuri) [reply .azure-pipelines/scripts/validate_samples.R]: sounds good, updated the script

**Pull request merged**